Include.psm1

```diff
@@ -499,7 +499,6 @@ function Set-Stat {
     }
 
     [PSCustomObject]@{ 
-        Name                  = [String]$Name
```

Comment:
Why?

Reply:
It is data duplication...

The name is stored in the file name.

Reply:
> it is data duplication...

Yes, but why then do you store the name as an separate property in other stats structures, e.g. 
https://github.com/MultiPoolMiner/MultiPoolMiner/blob/2990b8944ddc21a79a9ff7f0c2a25f467816f7aa/Include.psm1#L482
https://github.com/MultiPoolMiner/MultiPoolMiner/blob/2990b8944ddc21a79a9ff7f0c2a25f467816f7aa/Include.psm1#L546

Reply:
The function returns an array of objects as is common in PowerShell so the name needs to be in the object.

Reply:
> so the name needs to be in the object.

So why are you not consistent and have it everywhere?

Reply:
It's data duplication. You would then have two fields (the filename and the name in the file). 

I understand that I have put them in a hash table internally but that allows quick indexing and consistency with current code because I made `Stats` global (I know I shouldn't really expose it but writing a class with a static variable would take time and require major code changes). They are output from the function as an array so need the name and is quicker to retrieve if it already has the name in the object. 

Again, the stats variable is 'controlled' internal data and the stats folder is not.